common/navbar.js

```diff
@@ -253,12 +297,20 @@
 
                     scope.onToggle = function (open, menuObject) {
                         if (open) {
+                            // when menu opens, calculate height needed
                             logService.logClientAction({
                                 action: logService.getActionString(logService.logActions.NAVBAR_MENU_OPEN, "", ""),
                                 names: menuObject.names
                             });
                         }
+
                         onToggle(open);
+                        // unset height on dropdowns first
+                        var openDropdowns = document.querySelectorAll(".dropdown.open ul");
```

Comment:
Shouldn't this be part of `onToggle` function?

Reply:
Calling it here because of needing `$window.innerHeight` for the `checkHeight` function. $window is no defined in the scope for the `onToggle` function.

Reply:
You can pass the $window object to this function. I think it's cleaner to have all the functionality in the same function.